experimental/skills/DialogToDialog/DialogRootBot/Cards/welcomeCard.json

```diff
@@ -0,0 +1,128 @@
+﻿{
```

Comment:
This card needs some work, but not enough that it should hold up pushing the sample.

Reply:
OK, marking it as resolved, I can update it if you can think if a better way.

---
In reply to: [356209367](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356209367) [](ancestors = 356209367)